### ISSUE TITLE
rosbash: "find: paths must precede expression" on auto-complete attempt of rosmsg (with msg files in the CWD)

### ISSUE THREAD

**Issue by gavanderhoorn** [ros/ros#192]
To reproduce:

```
docker run -it --rm ros:melodic
source /opt/ros/melodic/setup.bash
roscd sensor_msgs/msg
rosmsg show sen<tab>
```

error returned then is:

```
find: paths must precede expression: `CameraInfo.msg'
find: possible unquoted pattern after predicate `-name'?
```

I've tried to understand what is going on in the auto-complete hook for `rosmsg`, but I've never had the pleasure of working with `compgen` so it's all black magic to me. It seems like the problem *could be* somewhere around:

https://github.com/ros/ros/blob/fd860146ff14efc425ef020ce570b3ba1e950772/tools/rosbash/rosbash#L902-L905

**Comment by gavanderhoorn** [comment]
I have a hunch that something like [this](https://stackoverflow.com/questions/27922449) is happening.

**Comment by jexner** [comment]
@gavanderhoorn Yes, that was the actual issue. #218 provides a fix.

**Comment by dirk-thomas** [comment]
Addressed by #218.
